resources/tentative/newssite/news-next/package.json

```diff
@@ -10,9 +10,11 @@
     "serve": "http-server ./dist -p 7002 -c-1 --cors"
   },
   "dependencies": {
+    "classnames": "^2.3.2",
     "eslint": "8.39.0",
     "eslint-config-next": "13.3.4",
     "http-server": "^14.1.1",
+    "news-site-css": "^1.6.5",
```

Comment:
I wonder if this should be a `file:` reference to a local directory in Speedometer as opposed to distributing through npm? I guess we do this already with todo CSS but I think it could be easier to audit and update styles over time if they live in this project. What do you think?

Reply:
I tinkered with that idea and it's an option.

Would the same approach be desirable, if we introduce global css changes to the todomvc apps? 
I'd like to find a consistent approach moving forward for other workloads as well. This seems like a step before moving towards something like a real monorepo, where we'd have better ways of managing dependencies and sharing packages.

The only downside (or change) in process I see with using locally linked packages, is that changes to these than would run through the same approval process as everything else in this repo... but maybe that's a good thing.

Reply:
> (i.e. if you intend to use this template as a tool for other projects, more like TodoMVC).

That was my initial thought, but I also don't want to hold up this pr. 
Before I make the change to a local package, @rniwa - do you have any preference?
Long term goal we could also discuss in an issue, where it might be more fitting.

Reply:
If we do intend to keep it as a npm package, it would be good to use a scoped package @speedometer.

Reply:
I like that idea, in that case if it's "official" who should be the owner? Is there a webkit npm account? 
@rniwa  - in general how would you like to manage css packages?

Reply:
Per today's discussion, we prefer having a local package.

Reply:
@rniwa - last push did just that. 
Please let me know if there's anything else to adjust or if we're good to merge 😄